lte/gateway/python/integ_tests/s1aptests/test_eps_bearer_context_status_def_bearer_deact.py

```diff
@@ -0,0 +1,240 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import ipaddress
+import time
+import unittest
+
+import gpp_types
+import s1ap_types
+import s1ap_wrapper
+from integ_tests.s1aptests.s1ap_utils import SpgwUtil
+
+
+class TestEpsBearerContextStatusDefBearerDeact(unittest.TestCase):
+    """Test default bearer deactivation with EPS bearer context status"""
+
+    def setUp(self):
+        """Initialize"""
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
+        self._spgw_util = SpgwUtil()
+
+    def tearDown(self):
+        """Cleanup"""
+        self._s1ap_wrapper.cleanup()
+
+    def test_eps_bearer_context_status_def_bearer_deact(self):
+        """Attach a single UE. Create 2 secondary PDNs and add 2
```

Comment:
Fix the comment, only 1 dedicated bearer is getting added for each secondary PDN in line 97-142. 
There are total of 5 bearers as seen in line 153.